AssertionError in production

As usual, some bots are sending HTTP GET requests to my server's catris port. But this time, that lead to an assertion failure...

```
Apr 26 21:51:05 catris python3[14929]: [INFO] (client 4) New connection
Apr 26 21:51:05 catris python3[14929]: [INFO] (client 4) There are now 1 connected clients
Apr 26 21:51:05 catris python3[14929]: [INFO] (client 4) Name asking done: 'GET / HTTP/1.1'
Apr 26 21:51:05 catris python3[14929]: [INFO] (client 4) Creating new lobby: 1LG1WF
Apr 26 21:51:05 catris python3[14929]: [INFO] (client 4) Joining lobby: 1LG1WF
Apr 26 21:51:05 catris python3[14929]: [INFO] (client 4) Joining a game with 0 existing players: <catris.games.ring.RingGame object at 0x7f18548d56a0>
Apr 26 21:51:05 catris python3[14929]: [ERROR] Task exception was never retrieved
Apr 26 21:51:05 catris python3[14929]: future: <Task finished name='Task-38171' coro=<Game._move_blocks_down_task() done, defined at /home/catris/catris/games/game_base_class.py:513> exception=AssertionErro>
Apr 26 21:51:05 catris python3[14929]: Traceback (most recent call last):
Apr 26 21:51:05 catris python3[14929]:   File "/home/catris/catris/games/game_base_class.py", line 518, in _move_blocks_down_task
Apr 26 21:51:05 catris python3[14929]:     await self.pause_aware_sleep(0.5 / (1 + self.get_duration_sec() / 600))
Apr 26 21:51:05 catris python3[14929]:   File "/home/catris/catris/games/game_base_class.py", line 70, in get_duration_sec
Apr 26 21:51:05 catris python3[14929]:     assert not self.is_paused
Apr 26 21:51:05 catris python3[14929]: AssertionError
```

Who needs fuzzers when bots spam your service anyway :DDD

--- a/catris/games/game_base_class.py
+++ b/catris/games/game_base_class.py
@@ -67,11 +67,7 @@ class Game:
         self.need_render_event.set()
 
     def get_duration_sec(self) -> float:
-        if self.is_paused:
-            end_time = self._last_pause_start
-        else:
-            end_time = time.monotonic_ns()
-
+        end_time = self._last_pause_start if self.is_paused else time.monotonic_ns()
         duration_ns = end_time - self._start_time - self._time_spent_in_pause
         return duration_ns / (1000 * 1000 * 1000)
 

--- a/catris/games/game_base_class.py
+++ b/catris/games/game_base_class.py
@@ -67,8 +67,12 @@ class Game:
         self.need_render_event.set()
 
     def get_duration_sec(self) -> float:
-        assert not self.is_paused
-        duration_ns = time.monotonic_ns() - self._start_time - self._time_spent_in_pause
+        if self.is_paused:
+            end_time = self._last_pause_start
+        else:
+            end_time = time.monotonic_ns()
+
+        duration_ns = end_time - self._start_time - self._time_spent_in_pause
         return duration_ns / (1000 * 1000 * 1000)
 
     async def pause_aware_sleep(self, sleep_time: float) -> None:

